pkg/handler/task_info.go

```diff
@@ -35,11 +35,11 @@ func RegisterTaskInfoRoutes(engine *echo.Group, daoReg *dao.DaoRegistry) {
 // @ID           listTasks
 // @Description  list tasks
 // @Tags         tasks
-// @Param		 offset query int false "Offset into the list of results to return in the response"
-// @Param		 limit query int false "Limit the number of items returned"
-// @Param		 status query string false "Filter tasks by status using an exact match"
-// @Param 		 type query string false "Filter tasks by type using an exact match"
-// @Param 		 repository_uuid query string false "Filter tasks by associated repository UUID using an exact match"
+// @Param		 offset query int false "Starting point for retrieving a subset of results. Determines how many items to skip from the beginning of the result set. Default value:`0`."
+// @Param		 limit query int false "Number of items to include in response. Use it to control the number of items, particularly when dealing with large datasets. Default value: `100`."
+// @Param		 status query string false "A comma separated list of statuses to control response. Statuses can include `running`, `completed`, `failed`."
+// @Param 		 type query string false "Filter results based on a specific task types. Helps to narrow down the results to a specific type. Task types can be `snapshot` or `introspect`. "
+// @Param 		 repository_uuid query string false "A unique identifier of a repository to filter the results. Note that you cannot interact with anything by using `GET`"
```

Comment:
```suggestion
// @Param 		 repository_uuid query string false "A unique identifier of a repository to filter the results.`"
```
("Note that you cannot interact with anything by using `GET" was my comment to the TW in the google doc)